### PR TITLE
add user and role models

### PR DESCRIPTION
**Pull request by bjmllr**

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+email "MyString"
+address "MyString"
+address2 "MyString"
+city "MyString"
+state "MyString"
+zip_code "MyString"
+emergency_contact_name "MyString"
+emergency_contact_phone "MyString"
+waiver_signature "MyString"
+photo_release "MyString"
```
Comment (r33beers) [inline spec/factories/users.rb]: nit: indentation

File: spec/views/users/edit.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                     :state => "MyString",
+                     :zip_code => "MyString",
+                     :emergency_contact_name => "MyString",
+                     :emergency_contact_phone => "MyString",
+                     :waiver_signature => "MyString",
+                     :photo_release => "MyString"
+                   ))
+  end
+
+  xit "renders the edit user form" do
```
Comment (r33beers) [inline spec/views/users/edit.html.erb_spec.rb]: typo? should this be ```it```

File: spec/views/users/index.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+               :zip_code => "Zip Code",
+               :emergency_contact_name => "Emergency Contact Name",
+               :emergency_contact_phone => "Emergency Contact Phone",
+               :waiver_signature => "Waiver Signature",
+               :photo_release => "Photo Release"
+             )
+           ])
+  end
+
+  xit "renders a list of users" do
```
Comment (r33beers) [inline spec/views/users/index.html.erb_spec.rb]: typo? ```it```?

File: spec/views/users/show.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                     :state => "State",
+                     :zip_code => "Zip Code",
+                     :emergency_contact_name => "Emergency Contact Name",
+                     :emergency_contact_phone => "Emergency Contact Phone",
+                     :waiver_signature => "Waiver Signature",
+                     :photo_release => "Photo Release"
+                   ))
+  end
+
+  xit "renders attributes in <p>" do
```
Comment (r33beers) [inline spec/views/users/show.html.erb_spec.rb]: typo again?

File: db/migrate/20150731225751_create_roles.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class CreateRoles < ActiveRecord::Migration
```
Comment (r33beers) [inline db/migrate/20150731225751_create_roles.rb]: ```bundle exec rake db:setup``` fails, since there are two create_roles migrations

**Comment by r33beers** [comment]
@bjmllr can you swap out all of the static text in the views and replace it with message properties in the en.yml file? ignore es.yml for now.

**Comment by bjmllr** [comment]
need to convert the .erb to .slim

File: db/migrate/20150801001302_user_belongs_to_role.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+class UserBelongsToRole < ActiveRecord::Migration
+  def change
+    change_table :users do |t|
+      t.integer :role_id
```
Comment (r33beers) [inline db/migrate/20150801001302_user_belongs_to_role.rb]: ```t.belongs_to :role``` instead?

**Pull request merged**